src/k8s/pkg/k8sd/app/hooks_bootstrap.go

```diff
@@ -280,10 +281,13 @@ func (a *App) onBootstrapControlPlane(ctx context.Context, s state.State, bootst
 
 	switch cfg.Datastore.GetType() {
 	case "k8s-dqlite":
+		// NOTE: Default certificate expiration is set to 20 years.
+		defaultDuration := int(time.Now().AddDate(20, 0, 0).Sub(time.Now()) / time.Second)
```

Comment:
So, the main idea of this csr stuff is to implement a way to easily refresh certificates. That way, we can have those certificates expire early. 
I would expect the default to be way lower (a year?) or am I missing something here?

Reply:
`k8s-dqlite` certificate cannot be refreshed like this, as we need to coordinate this across the whole cluster.